src/conn/conn_sweep.c

```diff
@@ -55,6 +60,11 @@ __sweep_mark(WT_SESSION_IMPL *session, uint64_t now)
         dhandle->timeofdeath = now;
         WT_STAT_CONN_INCR(session, dh_sweep_tod);
     }
+    if (timer) {
+        time_stop = __wt_clock(session);
+        time_diff = WT_CLOCKDIFF_US(time_stop, time_start);
+        WT_STAT_CONN_INCRV(session, cache_sweep_idle_mark_time, time_diff);
```

Comment:
My opinion is that we can just do an increment instead of a timer here. It is more valuable to know how many handles are being sweeped rather than the total time every handle has taken.

Reply:
@jiechenbo, are you sure about this? Didnt we want to know if closing a dhandle would take more time because we are waiting on a lock? Both stats are valuable in my opinion.

Reply:
@jiechenbo and @etienneptl what is the consensus about these timer stats? If I have not heard by tomorrow, I will put them back in so this can be finished.

Reply:
@tod-johnson-mongodb, please allow an extra day as @jiechenbo was OOO.

Reply:
OK. Please the two of you resolve this as soon as possible.

Reply:
Hi @tod-johnson-mongodb, I discussed with @jiechenbo and we don't have to implement the timers in this ticket, sorry for the delay. The way the code is now is good.